src/MongoDB.Driver.Core/Core/Operations/RetryabilityHelper.cs

```diff
@@ -96,24 +100,36 @@ public static void AddRetryableWriteErrorLabelIfRequired(MongoException exceptio
             }
         }
 
+        public static bool IsNetworkException(Exception exception)
+        {
+            return __networkExceptions.Contains(exception.GetType());
```

Comment:
Does this include subclasses of  `MongoConnectionException`?  Should it?

Reply:
Changed according to discussion in the slack